iFixit/react-commerce
# Add clear button to collection search input

This PR closes #61 a fixes a bug on the collection state

## QA

1. Visit the Vercel PR preview
2. Visit any collection that contains products
3. Verify that a clear button appears when you type something on the collection search input
